### ISSUE TITLE
sample yaml paths

### ISSUE THREAD

**Issue by nsheff** [pepkit/looper#305]
in the current docs, the parameters for all the included plugins look like they specify the *folder* of where the yaml will be stored, but not the *filename*. Shouldn't the customizable path be to the filename rather than just a folder? otherwise, how do you know/customize the actual path?

For instance it says:

"If the parameter is not provided, the file will be saved in `{looper.output_dir}/submission`."

It should instead say: " If the parameter is not provided, the file will be saved in `{looper.output_dir}/submission/{sample.sample_name}.yaml`.

Instead of:

```yaml
pipeline_type: sample
var_templates:
  main: "{looper.piface_dir}/pipelines/pipeline1.py"
  sample_yaml_path: "{looper.output_dir}/custom_sample_yamls"
```

it should say:

```yaml
pipeline_type: sample
var_templates:
  main: "{looper.piface_dir}/pipelines/pipeline1.py"
  sample_yaml_path: "{looper.output_dir}/custom_sample_yamls/{sample.sample_name}.yaml"
```

right?

**Comment by stolarczyk** [comment]
well, the docs represent actual plugins behavior. But I agree, we should make this more flexible. I'll make the relevant code changes.

**Comment by stolarczyk** [comment]
resolved in 46722c3cc3a349f01b7da75cb214d2e75e344ad2

**Comment by nsheff** [comment]
thanks.
